### PR TITLE
Makes runtime station selectable by default

### PR DESCRIPTION
**Pull request by ninjanomnom**
Just a simple change to make using runtime station easier when testing.

### CODE REVIEW

**Comment by MrStonedOne** [comment]
make this the default map

**Comment by praisenarsie** [comment]
disable box while you're at it

**Comment by ninjanomnom** [comment]
No, the point of this is to make it easier to test things locally. If someone needs to test something on box they shouldn't have to change configs.

**Comment by praisenarsie** [comment]
it

was

a

joke

**Comment by Cyberboss** [comment]
I'd be against making it the default map purely because it lacks a full station testing environment and, surprise surprise, tends to runtime out the ass. Also lowers the chances of finding new startup runtimes that would occur on Box but not runtime quickly.

**Comment by ninjanomnom** [comment]
I don't care too much either way since people can change the map in game, but most changes people aren't looking for new runtimes in the stations. They usually just want to get in game and spawn their thing/system and test it works as intended.

**Comment by DoctoreMad** [comment]
REMEMBER THAT TIME WE ACCIDENTALLY THE WHOLE SERVER ON RUNTIME STATION

File: config/maps.txt [hunk 1]
```diff
@@ -14,7 +14,6 @@ Format:
 endmap
 
 map boxstation
-	default
```
Comment (lzimann) [inline config/maps.txt]: keep box as default

**Pull request merged**